### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by topeogunleye**
### Added Changes
- Added test for controllers

### CODE REVIEW

File: test/controllers/posts_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'test-helper'
```
Comment (medaminedev66) [inline test/controllers/posts_controller_test.rb:1]: - [ ] Following the project requirements, you should use RSpec gem for testing. Please add RSpec as a gem to your project and set up tests on the controllers using RSpec.

File: test/controllers/users_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'test-helper'
+
+# UsersControllersTest
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test 'should get index' do
```
Comment (medaminedev66) [inline test/controllers/users_controller_test.rb:5]: - [ ] Following the project requirments, you should create `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory with `type: request` as shown in the example code:
```ruby
 require 'rails_helper'

 RSpec.describe "Users", type: :request do
   ...
 end
```

**Review by medaminedev66 — COMMENTED** [review]
Hi @topeogunleye  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional
- There are no linter errors
- You have a descriptive PR

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Hi @topeogunleye, 👋 

## Highlights 👏 

- RSpec gem is used properly, well done!
- User and Post controllers are tested, good job!
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**